pkg/datapath/linux/config/config.go

```diff
@@ -440,12 +445,27 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 
 		if option.Config.EnableIPv4 {
 			nodePortIPv4Addrs := node.GetNodePortIPv4AddrsWithDevices()
-			ipv4 := byteorder.NetIPv4ToHost32(nodePortIPv4Addrs[directRoutingIface])
+			ip, ok := nodePortIPv4Addrs[directRoutingIface]
+			if !ok {
+				log.WithFields(logrus.Fields{
+					"nodePortIPv4Addrs":  nodePortIPv4Addrs,
```

Comment:
I'm not sure if it makes sense dumping a list of all addresses on a host, since this could potentially be large. This field could probably be removed.